package.json

```diff
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
+    "ember-ajax": "^2.5.6",
```

Comment:
I was experimenting. I believe that ember-ajax is the problem here, but at the same time mirage seems to depend on jquery implicitly. 
This problem is a several layers' cake, but I'll get to the bottom of it.